gpg-client-wrapper

```diff
@@ -9,7 +11,26 @@ while [[ $1 ]]; do
             # Keyserver options makes no sense for offline GPG VM, so it is
             # rejected by qubes-gpg-client and qubes-gpg-server. But since
             # it is forced by Torbirdy extension, simply ignore the option.
-            --keyserver-options) 
+            --keyserver-options)
+                shift 2
+                ;;
+            --yes)
+                shift
+                ;;
+            --compress-algo=*)
+                shift
+                ;;
+            --no-encrypt-to)
+                shift
+                ;;
```

Comment:
Those three options can be probably added to allowed options, instead of being ignored here.
Take a look at https://github.com/QubesOS/qubes-app-linux-split-gpg/blob/master/src/gpg-common.h - you need to add them to both gpg_allowed_options and gpg_long_options. For options without short version, you need add it also to enum at the beginning.